ui/app/src/assets/uiConfigDefaults.ts

```diff
@@ -97,10 +97,11 @@ const uiConfigDefaults: Pick<GlobalState['uiConfig'], 'preview' | 'launcher'> =
                 configuration: {
                   id: 'Pages',
                   label: 'Pages',
-                  icon: {
-                    baseClass: 'fa',
-                    expandedClass: 'fa-file-text-o',
-                    collapsedClass: 'fa-file-text'
+                  expandedIcon: {
+                    id: '@material-ui/icons/DescriptionOutlined'
+                  },
+                  collapsedIcon: {
+                    id: '@material-ui/icons/DescriptionRounded'
                   },
```

Comment:
This was mostly for testing purposes. Let's just leave `icon: { DescriptionOutlined }`